kinder/ci/workflows/skew-kubelet-1.23-on-1.24.yaml

```diff
@@ -0,0 +1,15 @@
+# AUTOGENERATED by https://git.k8s.io/kubeadm/kinder/ci/tools/update-workflows
```

Comment:
This does not seem to be about dryrun？

Reply:
Not familiar with the `./hack/update-workflows.sh` script, and the v1.23 related skew are generated. Is there a flag or place to configure it?